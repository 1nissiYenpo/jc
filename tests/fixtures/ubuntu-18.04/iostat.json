[{"percent_user":28.59,"percent_nice":11.11,"percent_system":33.13,"percent_iowait":1.52,"percent_steal":0.0,"percent_idle":25.66,"type":"cpu"},{"device":"loop0","tps":0.36,"kb_read_s":1.18,"kb_wrtn_s":0.0,"kb_read":108,"kb_wrtn":0,"type":"device"},{"device":"loop1","tps":94.34,"kb_read_s":108.57,"kb_wrtn_s":0.0,"kb_read":9923,"kb_wrtn":0,"type":"device"},{"device":"loop2","tps":0.98,"kb_read_s":15.27,"kb_wrtn_s":0.0,"kb_read":1396,"kb_wrtn":0,"type":"device"},{"device":"loop3","tps":0.5,"kb_read_s":3.73,"kb_wrtn_s":0.0,"kb_read":341,"kb_wrtn":0,"type":"device"},{"device":"loop4","tps":0.57,"kb_read_s":11.6,"kb_wrtn_s":0.0,"kb_read":1060,"kb_wrtn":0,"type":"device"},{"device":"loop5","tps":0.36,"kb_read_s":1.18,"kb_wrtn_s":0.0,"kb_read":108,"kb_wrtn":0,"type":"device"},{"device":"loop6","tps":0.36,"kb_read_s":1.18,"kb_wrtn_s":0.0,"kb_read":108,"kb_wrtn":0,"type":"device"},{"device":"loop7","tps":0.58,"kb_read_s":11.51,"kb_wrtn_s":0.0,"kb_read":1052,"kb_wrtn":0,"type":"device"},{"device":"fd0","tps":0.97,"kb_read_s":3.89,"kb_wrtn_s":0.0,"kb_read":356,"kb_wrtn":0,"type":"device"},{"device":"sda","tps":179.87,"kb_read_s":7918.74,"kb_wrtn_s":9915.54,"kb_read":723773,"kb_wrtn":906280,"type":"device"},{"device":"scd0","tps":11.58,"kb_read_s":260.7,"kb_wrtn_s":0.0,"kb_read":23828,"kb_wrtn":0,"type":"device"},{"device":"scd1","tps":11.58,"kb_read_s":258.69,"kb_wrtn_s":0.0,"kb_read":23644,"kb_wrtn":0,"type":"device"},{"device":"loop8","tps":138.52,"kb_read_s":149.51,"kb_wrtn_s":0.0,"kb_read":13665,"kb_wrtn":0,"type":"device"},{"device":"loop9","tps":0.57,"kb_read_s":3.76,"kb_wrtn_s":0.0,"kb_read":344,"kb_wrtn":0,"type":"device"},{"device":"loop10","tps":1.01,"kb_read_s":5.02,"kb_wrtn_s":0.0,"kb_read":459,"kb_wrtn":0,"type":"device"},{"device":"loop11","tps":0.36,"kb_read_s":1.18,"kb_wrtn_s":0.0,"kb_read":108,"kb_wrtn":0,"type":"device"},{"device":"loop12","tps":0.45,"kb_read_s":3.68,"kb_wrtn_s":0.0,"kb_read":336,"kb_wrtn":0,"type":"device"},{"device":"loop13","tps":0.39,"kb_read_s":3.62,"kb_wrtn_s":0.0,"kb_read":331,"kb_wrtn":0,"type":"device"},{"device":"loop14","tps":0.58,"kb_read_s":3.81,"kb_wrtn_s":0.0,"kb_read":348,"kb_wrtn":0,"type":"device"},{"device":"loop15","tps":0.59,"kb_read_s":3.75,"kb_wrtn_s":0.0,"kb_read":343,"kb_wrtn":0,"type":"device"}]
