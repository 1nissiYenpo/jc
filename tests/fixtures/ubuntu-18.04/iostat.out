Linux 4.15.0-158-generic (kbrazil-ubuntu) 	12/01/2021 	_x86_64_	(1 CPU)

avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          28.59   11.11   33.13    1.52    0.00   25.66

Device             tps    kB_read/s    kB_wrtn/s    kB_read    kB_wrtn
loop0             0.36         1.18         0.00        108          0
loop1            94.34       108.57         0.00       9923          0
loop2             0.98        15.27         0.00       1396          0
loop3             0.50         3.73         0.00        341          0
loop4             0.57        11.60         0.00       1060          0
loop5             0.36         1.18         0.00        108          0
loop6             0.36         1.18         0.00        108          0
loop7             0.58        11.51         0.00       1052          0
fd0               0.97         3.89         0.00        356          0
sda             179.87      7918.74      9915.54     723773     906280
scd0             11.58       260.70         0.00      23828          0
scd1             11.58       258.69         0.00      23644          0
loop8           138.52       149.51         0.00      13665          0
loop9             0.57         3.76         0.00        344          0
loop10            1.01         5.02         0.00        459          0
loop11            0.36         1.18         0.00        108          0
loop12            0.45         3.68         0.00        336          0
loop13            0.39         3.62         0.00        331          0
loop14            0.58         3.81         0.00        348          0
loop15            0.59         3.75         0.00        343          0

